shared/cfg/guest-os/Linux/Fedora/23.ppc64le.cfg

```diff
@@ -0,0 +1,19 @@
+- 23.ppc64le:
+    image_name = images/f23-ppc64le
+    vm_arch_name = ppc64le
+    os_variant = fedora23
+    no unattended_install..floppy_ks
+    boot_path = ppc/ppc64
+    unattended_file_kernel_param_name = ks
+    unattended_install, svirt_install:
+        kernel_params = 'repo=cdrom:/dev/disk/by-label/Fedora-Server-23-ppc64le'
```

Comment:
I gave it yet another try and the `repo=cdrom:...` breaks it. Without that the installation works smoothly, with it it always fails, the serial output looks like this:

```
2016-06-07 13:03:05: [    9.441875] xor: using function: altivec (22470.400 MB/sec)
2016-06-07 13:03:05: [    9.443453] md: raid6 personality registered for level 6
2016-06-07 13:03:05: [    9.443492] md: raid5 personality registered for level 5
2016-06-07 13:03:05: [    9.443525] md: raid4 personality registered for level 4
2016-06-07 13:03:05: [    9.449031] md: raid10 personality registered for level 10
2016-06-07 13:03:05: [    9.450296] md: linear personality registered for level -1
2016-06-07 13:03:05: [    9.456969] device-mapper: multipath round-robin: version 1.0.0 loaded
2016-06-07 13:03:05: [    9.475626] RPC: Registered named UNIX socket transport module.
2016-06-07 13:03:05: [    9.475703] RPC: Registered udp transport module.
2016-06-07 13:03:05: [    9.475736] RPC: Registered tcp transport module.
2016-06-07 13:03:05: [    9.475768] RPC: Registered tcp NFSv4.1 backchannel transport module.
2016-06-07 13:03:05: [^[[32m  OK  ^[[0m] Started dracut pre-udev hook.
2016-06-07 13:03:05: [    9.558058] audit: type=1130 audit(1465318984.610:6): pid=1 uid=0 auid=4294967295 ses=4294967295 subj=kernel msg='unit=dracut-pre-udev comm="systemd" exe="/usr/lib/systemd/systemd" hostname=? addr=? terminal=? res=success'
2016-06-07 13:03:05:          Starting udev Kernel Device Manager...
2016-06-07 13:03:05: [^[[32m  OK  ^[[0m] Started udev Kernel Device Manager.
2016-06-07 13:03:05: [    9.563811] audit: type=1130 audit(1465318984.620:7): pid=1 uid=0 auid=4294967295 ses=4294967295 subj=kernel msg='unit=systemd-udevd comm="systemd" exe="/usr/lib/systemd/systemd" hostname=? addr=? terminal=? res=success'
2016-06-07 13:03:05:          Starting dracut pre-trigger hook...
2016-06-07 13:03:05: [    9.636988] dracut-pre-trigger[442]: cat: /tmp/dd_disk: No such file or directory
2016-06-07 13:03:05: [^[[32m  OK  ^[[0m] Started dracut pre-trigger hook.
2016-06-07 13:03:05: [    9.655486] audit: type=1130 audit(1465318984.710:8): pid=1 uid=0 auid=4294967295 ses=4294967295 subj=kernel msg='unit=dracut-pre-trigger comm="systemd" exe="/usr/lib/systemd/systemd" hostname=? addr=? terminal=? res=success'
2016-06-07 13:03:05:          Starting udev Coldplug all Devices...
2016-06-07 13:03:05:          Mounting Configuration File System...
2016-06-07 13:03:05: [^[[32m  OK  ^[[0m] Mounted Configuration File System.
2016-06-07 13:03:05: [    9.697066] virtio-pci 0000:00:07.0: enabling device (0100 -> 0103)
2016-06-07 13:03:05: [    9.697553] virtio-pci 0000:00:07.0: virtio_pci: leaving for legacy driver
2016-06-07 13:03:05: [    9.698068] virtio-pci 0000:00:06.0: enabling device (0100 -> 0103)
2016-06-07 13:03:05: [    9.698558] virtio-pci 0000:00:06.0: virtio_pci: leaving for legacy driver
2016-06-07 13:03:05: [^[[32m  OK  ^[[0m] Started udev Coldplug all Devices.
2016-06-07 13:03:05: [    9.699022] audit: type=1130 audit(1465318984.750:9): pid=1 uid=0 auid=4294967295 ses=4294967295 subj=kernel msg='unit=systemd-udev-trigger comm="systemd" exe="/usr/lib/systemd/systemd" hostname=? addr=? terminal=? res=success'
2016-06-07 13:03:05: [^[[32m  OK  ^[[0m] Reached target System Initialization.
2016-06-07 13:03:05: [^[[32m  OK  ^[[0m] Reached target Basic System.
2016-06-07 13:03:05:          Starting dracut initqueue hook...
2016-06-07 13:03:05: [    9.702999] virtio-pci 0000:00:05.0: enabling device (0100 -> 0103)
2016-06-07 13:03:05: [    9.703599] virtio-pci 0000:00:05.0: virtio_pci: leaving for legacy driver
2016-06-07 13:03:05: [    9.706758] virtio-pci 0000:00:03.0: enabling device (0100 -> 0103)
2016-06-07 13:03:05: [    9.707679] virtio-pci 0000:00:03.0: virtio_pci: leaving for legacy driver
2016-06-07 13:03:05: [    9.758443] scsi host0: Virtio SCSI HBA
2016-06-07 13:03:05: [    9.758822] scsi 0:0:0:0: CD-ROM            QEMU     QEMU CD-ROM      2.5+ PQ: 0 ANSI: 5
2016-06-07 13:03:05: [    9.759027] scsi 0:0:1:0: CD-ROM            QEMU     QEMU CD-ROM      2.5+ PQ: 0 ANSI: 5
2016-06-07 13:03:05: [    9.769855] sr 0:0:0:0: [sr0] scsi3-mmc drive: 16x/50x cd/rw xa/form2 cdda tray
2016-06-07 13:03:05: [    9.769916] cdrom: Uniform CD-ROM driver Revision: 3.20
2016-06-07 13:03:05: [    9.770130] sr 0:0:0:0: Attached scsi generic sg0 type 5
2016-06-07 13:03:05: [    9.770342] sr 0:0:1:0: [sr1] scsi3-mmc drive: 16x/50x cd/rw xa/form2 cdda tray
2016-06-07 13:03:05: [    9.770548] sr 0:0:1:0: Attached scsi generic sg1 type 5
2016-06-07 13:03:05: [    9.783546] virtio_net virtio1 enp0s6: renamed from eth0
2016-06-07 13:03:06: [   10.892634] dracut-initqueue[527]: cat: /tmp/dd_disk: No such file or directory
2016-06-07 13:03:27: [   31.474159] random: nonblocking pool is initialized
2016-06-07 13:33:16: (Process terminated with status 0)
```

Anaconda fail to start, because it fails to find the ks so it simply waits there until the inactivity watcher kills it.

Did you try it without the `repo`? Does it fail for you or what is the reason you want that included? With `kernel_params = "ks=cdrom nicdelay=60 console=hvc0 console=ttyS0 serial"` it works flawlessly for me.